libs/platform/src/lib/components/button/button.component.ts

```diff
@@ -32,18 +39,14 @@ export class ButtonComponent {
     @Input()
     disabled: boolean;
 
-    /** The type of the button. Types include 'standard', 'positive', 'medium', and 'negative'.
-     * Leave empty for default (Action button).'*/
+    /** The type of the button. Types include 'standard','positive' and 'negative' etc.
+     * Leave empty for default (standard button).'*/
     @Input()
     type: ButtonType;
 
     @Input()
     fdType: ButtonType;
```

Comment:
fdType can be removed, as only type is used in template.

Reply:
Done